vllm/distributed/parallel_state.py

```diff
@@ -1147,6 +1147,16 @@ def get_tensor_model_parallel_rank():
     return get_tp_group().rank_in_group
 
 
+def get_pipeline_model_parallel_world_size():
```

Comment:
the recommended way is to use `get_pp_group().world_size` directly. actually we should refactor `get_tensor_model_parallel_world_size` , but we deferred it because the change is too large.